packages/genesys-spark-components/src/components/beta/gux-calendar-beta/components/single/gux-calendar.tsx

```diff
@@ -272,7 +272,10 @@ export class GuxCalendar {
           })}
           onClick={() => this.setNewFocusedMonth(-1)}
         >
-          <gux-icon decorative icon-name="chevron-small-left"></gux-icon>
+          <gux-icon
+            decorative
+            icon-name="custom/chevron-left-small-regular"
```

Comment:
'chevron-small-left' is not listed in the deprecated names but 'chevron-small-right' is. How come?

Reply:
Fixed